cmd/thanos/compact.go

```diff
@@ -32,7 +28,7 @@ func registerCompact(m map[string]setupFunc, app *kingpin.Application, name stri
 		Hidden().Default("true").Bool()
 
 	httpAddr := cmd.Flag("http-address", "Listen host:port for HTTP endpoints.").
```

Comment:
what do you think of having function in `flags.go`: `func httpAddrFlag() string { return cmd.Flag(...) }`, so that you don't have to repeat it in `registerServerFlags` function too?

Reply:
... but only the common flags, goal of this suggestion is to increase DRYness

Reply:
Think if we do it again we should look into it not too bothered till then TBH ... but agree it would help with defaults and flag text.

Reply:
like the idea